SU-SWS/stanford_capx
# CAPX-94: Updated CAPX info on supported entity edit forms.

How to test:
- Import some profiles
- Edit any imported profile entity
- Check "CAPx Information" vertical tab/fieldset
